packages/@react-spectrum/datepicker/test/DateField.test.js

```diff
@@ -664,4 +664,75 @@ describe('DateField', function () {
       });
     });
   });
+
+  describe('style', () => {
```

Comment:
I think all of these tests can be covered by chromatic

these unit tests are likely to break in unhelpful ways, as a change to the styles could result in the correct rendering and behavior, but break here